packages/customer-effort-score/src/style.scss

```diff
@@ -3,6 +3,6 @@
 	top: 300px;
 	left: 300px;
 	background: #faa;
-	z-index: 999;
+	z-index: 9999;
```

Comment:
Is this relative to something?  Maybe we could add a comment describing what it needs to be above/below.

Reply:
We should take care of this in a more robust fashion in #5515, so this is just a temporary fix.

Reply:
This is just a temporary modal style, it will be removed soon in #5515